runtime/bin/tapasco-build-libs

```diff
@@ -70,6 +76,18 @@ parser.add_argument(
     '--enable_svm',
     help='enable driver support for Shared Virtual Memory (SVM)',
     action='store_true')
+parser.add_argument(
+    '--enable_sim',
+    nargs='?',
+    help='Enable simulation support in the driver and setup the tapasco simulation. Selecting runtime will only compile the runtime with simulation support and skip setup of the simulation installation.',
+    default=None,
+    choices=[
+        '',
```

Comment:
Would you mind adding a worded choice instead of the empty string? It could be something like ``full`` or ``toolflow``. I personally had to do some trial and error to figure out that I need to pass ``--enable_sim=''`` on the simulation PC.

Reply:
I just tested it with only `--enable_sim` and it works for me. Specifying nothing will compile the simlibs and the runtime, using  `--enable_sim=runtime` will only compile the tapasco runtime and kernel driver, so the host where the host-software runs doesnt need to have questa and vivado installed.

Reply:
Ok, could you add an option to only build the simlibs without the runtime, so you don't need to install Rust if you use separate machines for simulation and host.

Reply:
Nevermind, you can skip runtime build with ``--skip_runtime``.